README.md

```diff
@@ -112,14 +111,27 @@ with `DataIntegrityProof` proof type using the `ecdsa-rdfc-2019`,
 To add your implementation to this test suite, you will need to add 2 endpoints
 to your implementation manifest.
 - A credential issuer endpoint (`/credentials/issue`) in the `issuers` property.
+  - An optional id property specifying the issuer can be set alongside the endpoint.
+  - If provided, the specified issuer id will be used as the issuer property on verifiable credentials
+    in the tests.
+  - The issuer id should be easy for other implementers to dereference.
+    - It is recommended the issuer id be `did:key`.
```

Comment:
```suggestion
  - An optional `id` property may be set alongside the `endpoint`.
  - If provided, the specified `issuer.id` will be added to Verifiable Credentials
    in the tests.
  - If present, the `issuer.id` MUST use the `did:key` method and MUST dereferenceable.
```

Reply:
Simplifying and clarifying. Pretty sure the `id` values MUST be dereferenceable--and that the test suite only currently supports `did:key` making both those things requirements, not recommendations.